adafruit_platformdetect/board.py

```diff
@@ -349,6 +351,15 @@ def _stm32mp1_id(self) -> Optional[str]:
             return boards.OSD32MP1_BRK
         if "OSD32MP1-RED" in board_value:
             return boards.OSD32MP1_RED
+        if "STM32MP1XX OLinuXino" in board_value:
+            return boards.STMP157 
+        return None
+
+    def _stm32mp157_id(self) -> Optional[str]:
```

Comment:
The logic in here should just be able to be added to `_stm32mp1_id()`

Reply:
done